apps/studio/src/server/modules/auth/auth.service.ts

```diff
@@ -30,10 +30,6 @@ export const verifyToken = async (
       throw new VerificationError("Token is invalid or has expired")
     }
 
-    if (verificationToken.attempts > 5) {
```

Comment:
this one also cannot kill. will get new vuln due to bruteforcing haha

Reply:
This got removed cos it's redundant, ther s an identical block above oddly enough HAHA